src/test/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/ETV1ContainerTest.groovy

```diff
@@ -35,4 +38,24 @@ class ETV1ContainerTest extends ETContainerTest {
                 .withLogConsumer(new Slf4jLogConsumer(LOGGER))
                 .waitingFor(Wait.forHttp("/api/v1/live"))
     }
+
+    def "Perform provide logs step with reports"() {
+        given: "a test execution pipeline"
+            String script = """
+            node {
+                withEnv(['ET_API_HOSTNAME=${etContainer.host}', 'ET_API_PORT=${etContainer.getMappedPort(ET_PORT)}']) {
+                    ttRunPackage testCasePath: 'test.pkg'
+                    ttProvideLogs()
+                }
+            }
+            """.stripIndent()
+            WorkflowJob job = jenkins.createProject(WorkflowJob.class, "pipeline")
+            job.setDefinition(new CpsFlowDefinition(script, true))
+        when: "scheduling a new build"
+            WorkflowRun run = jenkins.buildAndAssertStatus(Result.FAILURE, job)
+
+        then: "expect successful test completion"
+            jenkins.assertLogContains("Providing ecu.test logs to jenkins.", run)
+            jenkins.assertLogContains("Downloading report folders is not supported by api v1. Use ecu.test > 2024.2 with api v2 instead.", run)
```

Comment:
`>=` ???
The use do not care about the api version and is not able to change it. This is our own handling in the background. Therefor i would delete the api information